frontend/src/components/ApartmentCard/ApartmentCard.tsx

```diff
@@ -1,24 +1,31 @@
 import React, { ReactElement } from 'react';
-import ApartmentImg from '../../assets/apartment-sample.png';
+import ApartmentImg from '../../assets/apartment-placeholder.png';
```

Comment:
Great job!! The last thing is figuring out how to make the cards not get too big vertically, i.e. restraining the height of the cards while responsively sizing the width. Tbh my frontend/css knowledge isn't that extensive, so we can play around with it and see what we can do